jest.unit.config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
```

Comment:
Is it possible for these jest config files to be typescript? Also is there some way to pull out the shared config between the two of them? Even just a base config file that gets imported

Reply:
The current test files do not support typescript config. I created a branch `dev/ts-jest-config` with the config files in typescript, but many syntax errors within tests need to be resolved to accommodate typescript configuration for future reference.